src/kv/flatbufferwrapper.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include <data_generated.h>
```

Comment:
If we record the output of `flatc` not in `build/` (either in `build/flatbuffers/` or directly in `src/flatbuffers`), then it would be clearer where this file comes from.

Reply:
`src/flatbuffers` is suboptimal since we will then have to tell git to explicitly not commit those files and I don't think generated files should go under `src`. `build/flatbuffers` is an option to keep the `build` dir clean so we can think about that, I don't have a string opinion either way.

Reply:
I second generating in a build subfolder, preferably 2 layers deep so we can a) only add a subdir to the include directories, and b) clearly identify the generated headers.

eg - if this is `build/generated/flatbuffers/data_generated.h` then the cmake only adds `build/generated` to the include dirs, and this is `#include "flatbuffers/data_generated.h"`.

Reply:
fwiw, I think the build directory is for scripts, not for people, that's what install is for. I'm fine with subdirs, but I'm also fine without.